### ISSUE TITLE
Why do we need detach the level box in ROIAlign?

### ISSUE THREAD

**Issue by Superlee506** [multimodallearning/pytorch-mask-rcnn#5]
![image](https://user-images.githubusercontent.com/13114675/37560030-07ad422a-2a07-11e8-9a29-412a727a0064.png)

As in [Line 463](https://github.com/multimodallearning/pytorch-mask-rcnn/blob/master/model.py#L463), why do we need to stop gradient propogation to ROI proposals?

**Comment by lasseha** [comment]
We do not need to, training just seems to be more stable when stopping the gradient flow through the proposal coordinates. See [here](https://stackoverflow.com/questions/46561390/4-step-alternating-rpn-faster-r-cnn-training-tensorflow-object-detection-mo) for a similar question on stackoverflow.
